framework/components/AButtonCard/AButtonCard.mdx

```diff
@@ -0,0 +1,90 @@
+---
+name: Button Card
+route: /components/buttonCard
+components: AButtonCard
+---
+
+import {Playground, Props} from "../../../docs";
+
+# Button Cards
+
+## Install
+
+Integrate with your build to [auto-import](/#integrating), or add an import in your component:
+
+```
+import AButtonCard from "@cisco-ats/atomic-react/AButtonCard";
+// or if you're able to treekshake node_modules
```

Comment:
These two lines can be removed, since 3.0.0 importing via `@cisco-ats/atomic-react/AComponentName` has been removed